conf/app.routes

```diff
@@ -1,13 +1,13 @@
 # microservice specific routes
 
-POST        /verification-requests                     uk.gov.hmrc.emailverification.controllers.EmailVerificationController.requestVerification
-POST        /verified-email-addresses                  uk.gov.hmrc.emailverification.controllers.EmailVerificationController.validateToken
-POST        /verified-email-check                      uk.gov.hmrc.emailverification.controllers.EmailVerificationController.verifiedEmail
+POST        /verification-requests                     uk.gov.hmrc.emailverification.controllers.EmailVerificationController.requestVerification()
```

Comment:
Please can you place at the top:
POST /verify-email xxxxx
GET /verification-status/:credId xxxx
... at the top of the file.

Next, add a comment above all the others that says "DEPRECATED endpoints are below:"